mindjunkies/courses/tests/test_views.py

```
@@ -0,0 +1,118 @@
+    client.force_login(user)
+    url = reverse("rate_course", kwargs={"course_slug": course.slug})
+
+    response = client.post(url, {"rating": 5, "review": "Great!"})
+    assert response.status_code == 302
+    assert Rating.objects.filter(course=course, student=user).exists()
+
+
+def test_rating_create_view_existing(client):
+    user = baker.make("accounts.User")
+    course = baker.make(Course)
+    rating = baker.make(Rating, course=course, student=user, rating=3, review="Okay")
```

Ruff (0.8.2)

Local variable `rating` is assigned to but never used

Remove assignment to unused variable `rating`

(F841)